src/responseTypes.ts

```diff
@@ -55,7 +55,7 @@ interface CompleteStatusResponse extends BaseStatusResponse {
 export type GetStatusResponse = IncompleteStatusResponse | CompleteStatusResponse;
 
 export interface ExecutionResult {
-  rows: Record<string, string>[];
+  rows: Record<string, any>[];
```

Comment:
What about `unknown` rather than `any` as Tukan suggested?

Reply:
Yes that would be fine... I just don't know how to use it. Can I just put unknown here and everything will just be the same? Or do I have to do something additional?

Reply:
```suggestion
  rows: Record<string, unknown>[];
```